static/app/views/releases/list/releaseCard/releaseCardCommits.tsx

```diff
@@ -42,15 +42,18 @@ function ReleaseCardCommits({release, withHeading = true}: Props) {
   ].join(' ');
 
   return (
-    <div className="release-stats">
+    <ReleaseStatsContainer className="release-stats">
```

Comment:
I know there are no hard rules right now but I think this would read a lot better as:

```
<Flex align="center" className="release-stats">
```

because it makes jsx parsable without having to know how `ReleaseStatsContainer` is implemented

Reply:
Definitely. I need to retrain my brain to reach for flex, thanks for reminding me :)